README.md

```diff
@@ -57,11 +57,20 @@ To install them simply run in a terminal:
 python3 -m pip install -U -r requirements.txt
 ```
 
-#### 3. Prepare a base ROM
+#### 3. Install pigment64
+
+Building also requires a rust crate called pigment64. To install rust and the crate, run these commands:
+
+```bash
+curl https://sh.rustup.rs -sSf | sh
+cargo install pigment64
+```
```

Comment:
pigment64 has prebuilt binaries, so instead of installing it systemwide we could just download those binaries like we do with ido.
This would allow us to avoid adding Rust as a dependency (I'm not opposed to adding it as a dependency, but some others may).

Reply:
Ok. I'll do that in the asset extraction pr and remove this from the readme.